### ISSUE TITLE
TabView status?

### ISSUE THREAD

**Issue by chucker** [lepoco/wpfui#847]
### Is your feature request related to a problem? Please describe

WinUI's TabView has features the WPF TabControl lacks:

![image](https://github.com/lepoco/wpfui/assets/20194/9f0216ca-7cc4-4154-8601-50697b3d8df4)

For example,

* tabs can be dragged to change their order
* …and they can be dragged _between windows_, or to create a new window
* tabs feature a close button
* there's an add button to the right

I would argue despite the similar visual metaphor, the TabView and TabControl are two different things. The TabView holds entire documents (for example, Windows 11's Notepad); the TabControl gives different views for a single document (for example, the General / Security / etc. tabs in a File Explorer's Properties dialog).

### Describe the solution you'd like

There's an existing WPF library that does most (all?) of the above in https://github.com/ButchersBoy/Dragablz, and you can see that in action in a modern UI in, for example, [the ModernWpf integration](https://github.com/Kinnara/ModernWpf/wiki/Dragablz-Integration).

I wonder if, in lieu of rolling an implementation of its own, there could be a `Wpf.Ui.Dragablz` library that wraps Dragablz, and adds WpfUi styling.

**Comment by tonyrichter** [comment]
Any news on this topic? We would really like that idea

**Comment by J9k9xtcejpamesgsxfki5Knepp** [comment]
yes,i want it too

**Comment by J9k9xtcejpamesgsxfki5Knepp** [comment]
> 是的，我也想要它

and if them merge to navigationView is best

**Comment by p6laris** [comment]
Yeah, the recent one TabControl doesn't have close neither the add button, which are essential. as @chucker mentioned they are different, but having a TabView would be nice.
